core/statistics/softwareVersion/softwareVersion.go

```diff
@@ -18,6 +19,7 @@ type tagVersion struct {
 	TagVersion string `json:"tag_name"`
 }
 
+// SoftwareVersionChecker is component used to check if a new software stable tag in available
```

Comment:
// SoftwareVersionChecker is a component which is used to check if a new software stable tag is available

Reply:
done.